app/modules/ads/module.js

```diff
@@ -52,9 +52,11 @@ class Ads {
     const { bidders } = window.Wikia.adBidders;
     const { krux } = window.Wikia.adServices;
 
+    this.biddersCalled = false;
+
     biddersDelay.resetPromise();
     bidders.requestBids({
-      responseListener: biddersDelay.markAsReady,
+      responseListener: biddersDelay.markAsReady.bind(this),
```

Comment:
To avoid this confusion -- what about:
```
responseListener: () => {
  biddersDelay.markAsReady();
  this.biddersCalled = true;
}
```

Reply:
...and I don't know why I didn't though about that :D Changed.